deployments/kubehound/notebook/KubehoundDSL_101.ipynb

```diff
@@ -22,49 +22,9 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "set notebook config to:\n",
-      "{\n",
-      "  \"host\": \"kubegraph\",\n",
-      "  \"port\": 8182,\n",
-      "  \"proxy_host\": \"\",\n",
-      "  \"proxy_port\": 8182,\n",
-      "  \"ssl\": false,\n",
-      "  \"ssl_verify\": true,\n",
-      "  \"sparql\": {\n",
-      "    \"path\": \"\"\n",
-      "  },\n",
-      "  \"gremlin\": {\n",
-      "    \"traversal_source\": \"g\",\n",
-      "    \"username\": \"\",\n",
-      "    \"password\": \"\",\n",
-      "    \"message_serializer\": \"graphsonv3\"\n",
-      "  },\n",
-      "  \"neo4j\": {\n",
-      "    \"username\": \"neo4j\",\n",
-      "    \"password\": \"password\",\n",
-      "    \"auth\": true,\n",
-      "    \"database\": null\n",
-      "  }\n",
-      "}\n"
-     ]
-    },
-    {
-     "data": {
-      "text/plain": [
-       "<graph_notebook.configuration.generate_config.Configuration at 0xffff67e61ac0>"
-      ]
-     },
-     "execution_count": 1,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "metadata": {
```

Comment:
Looking at the next lines (not in the diff so cannot comment on them), it seems like there is still a "connection" to the kubegraph cell.
Can we also delete the next cell?

Even if we don't, should we put the message you put elsewhere?
> `Connection is being initated directly from the docker using the env vars ...`

Reply:
Talked with Julien - this is expected to be, so we keep an example for users who want to know how it's done